image_client/picturae_csv_create.py

```diff
@@ -22,8 +22,17 @@
 
 
 class CsvCreatePicturae(Importer):
-    def __init__(self, date_string, istesting=False):
+    def __init__(self, date_string, istest = False):
         super().__init__(picturae_config, "Botany")
+        self.init_all_vars(date_string)
+
+        if istest is False:
+            self.run_all()
+
+
+    def init_all_vars(self, date_string):
```

Comment:
Don't know if this is a step in the right direction. Was trying to see if I could use this to initialize the class , without running self.run_all() in testing, but could be off.

Reply:
We'll leave it for now - I'll be in today and we can discuss